lib/shared/bucketing-assembly-script/assembly/bucketing/bucketing.ts

```diff
@@ -18,7 +18,15 @@ class BoundedHash {
     public bucketingHash: i32
 }
 
-export function generateBoundedHashes(user_id: string, target_id: string): BoundedHash {
+export function generateBoundedHashesFromJSON(user_id: string, target_id: string): string {
+    const boundedHash = _generateBoundedHashes(user_id, target_id)
+    const json = new JSON.Obj()
+    json.set('rolloutHash', boundedHash.rolloutHash)
+    json.set('bucketingHash', boundedHash.bucketingHash)
+    return json.stringify()
+}
+
+export function _generateBoundedHashes(user_id: string, target_id: string): BoundedHash {
```

Comment:
what are the underscore prefixes meant to represent?

Reply:
internal methods vs external ones.